### PR TITLE
D8 nid 607

### PR DESCRIPTION
**Pull request by neilblair**
Adds:
- basic image styles used across content types
- responsive image styles
- makes relevant responsive styles available in CKEditor
- news thumbnail display mode for media image entities
- alters the news teaser display to use the news thumbnail media display mode

TODO: need to look at mapping image styles from D7 to D8 in migration scripts. I was unable to match D7 machine names for image and responsive image styles in the D8 equivalents 😒

### CODE REVIEW

File: web/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Drupal editor configuration normalization
```
Comment (johangant) [inline web/.editorconfig:1]: Is this file needed? Looks like it was duplicated from the directory above `/web`?

File: web/.gitattributes [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# Drupal git normalization
```
Comment (johangant) [inline web/.gitattributes:1]: Is this file needed? Looks like it was duplicated from the directory above `/web`?

File: web/.editorconfig (hunk 1 above)
Comment (neilblair) [reply web/.editorconfig]: Yep not sure how that happened - removed it

File: web/.gitattributes (hunk 2 above)
Comment (neilblair) [reply web/.gitattributes]: Ditto - removed this one too

**Comment by neilblair** [comment]
Removed both of those files Johan

**Comment by neilblair** [comment]
Just added another commit - a tweak to image styles intended for news thumbnails - renamed them to be more generic "nigov landscape thumbnail" for better reuse.

**Comment by neilblair** [comment]
...And another commit to fix the new thumbnail responsive image style which got broken by the previous commit!
